documentation/catalog/reference/DAGs.md

```diff
@@ -32,36 +32,36 @@ The following are DAGs grouped by their primary tag:
 
 ### Data Refresh
 
-| DAG ID                                                        | Schedule Interval |
-| ------------------------------------------------------------- | ----------------- |
-| [`audio_data_refresh`](#audio_data_refresh)                   | `0 0 * * 1`       |
-| [`create_filtered_audio_index`](#create_filtered_audio_index) | `None`            |
-| [`create_filtered_image_index`](#create_filtered_image_index) | `None`            |
-| [`image_data_refresh`](#image_data_refresh)                   | `0 0 * * 1`       |
+| DAG ID                                                             | Schedule Interval |
+| ------------------------------------------------------------------ | ----------------- |
+| [`audio_data_refresh`](#media_type_data_refresh)                   | `0 0 * * 1`       |
+| [`create_filtered_audio_index`](#create_filtered_media_type_index) | `None`            |
+| [`create_filtered_image_index`](#create_filtered_media_type_index) | `None`            |
+| [`image_data_refresh`](#media_type_data_refresh)                   | `0 0 * * 1`       |
```

Comment:
It'd be nice if these were sorted by the mapped DAG ID, followed by the original DAG ID, so that related DAGs are always together. I guess in this version the DAGs are grouped by media type, but the difference in ordering for the audio and image DAGs reads a bit lazy/careless to me (especially imagining not having the context of this document being automatically generated).